### PR TITLE
feat(logger): use standard way

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description
standardization of logs

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [x] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Review by Lawrence-lkq — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/conf/default.py [hunk 1]
```diff
@@ -379,9 +380,30 @@
...
+else:
+    logging_to_console = False
+    # The dir allows both absolute and relative path, when it's relative, combine
+    # the value with project's base directory
+    logging_directory = Path(BASE_DIR) / Path(LOG_DIR)
+    logging_directory.mkdir(exist_ok=True)
+
+
+# 是否总是打印日志到控制台，默认关闭
+LOGGING_ALWAYS_CONSOLE = env.bool("LOGGING_ALWAYS_CONSOLE", True)
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/conf/default.py:402]: 这里应该是False?

File: src/dashboard/apigateway/apigateway/conf/default.py [hunk 2]
```diff
@@ -26,7 +27,7 @@
 from apigateway.common.env import Env
 from apigateway.conf.celery_conf import *  # noqa
 from apigateway.conf.celery_conf import CELERY_BEAT_SCHEDULE
-from apigateway.conf.log_utils import get_logging_config, makedirs_when_not_exists
+from apigateway.conf.log_utils import build_logging_config
 from apigateway.conf.utils import get_default_keepalive_options
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/conf/default.py:32]: 开启ruff扫描

File: src/dashboard/apigateway/apigateway/conf/default.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/conf/default.py:402]: done

File: src/dashboard/apigateway/apigateway/conf/default.py (hunk 2 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/conf/default.py:32]: done

**Review by wklken — COMMENTED** [review]
LGTM

**Review by wklken — APPROVED** [review]

**Pull request merged**